model/generators/oracles.py

```diff
@@ -27,13 +27,17 @@ class OracleRateGenerator(Generator):
     oracles_by_id: Dict[UUID, OracleProvider]
     oracles_by_pair: Dict[Pair, List[OracleProvider]]
     oracle_pairs: List[Pair]
+    rngp: RNGProvider
+    rng: np.random.Generator
```

Comment:
is this still used? Don't find usage

Reply:
You're right doesn't look like it's actually used in the generator, I deleted it 👍